### ISSUE TITLE
Add aria-hidden="true" to all elements (span) that insert an icon

### ISSUE THREAD

**Issue by Veyfeyken** [a-ui/core_branding_scss#146]
### Problem
[base/base.antwerpen-icons](https://a-ui.github.io/core_branding_scss/#base-antwerpen-icons)

Icon fonts inject an unknown character into the HTML though the CSS content property in combination with `:before` or `:after`. This character can be read by screenreaders and cause confusion.

```html
<span class="icon-a-card"></span>
```
```css
.icon-a-card:before {
	content: '\EA01';
}
```
### Solution

Add an `aria-hidden="true"` attribute on all elements (`<span>`) that insert an icon to prevent screenreaders from reading an unknown character.

### Links and articles

* [Understanding Success Criterion 1.1.1: Non-text Content
](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)
* [AnySurfer: Icon fonts (dutch)](https://www.anysurfer.be/nl/blog/detail/icon-fonts)
* [Font Awesome: Accessibility](https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility)

**Comment by TriangleJuice** [comment]
Fixed in #178

**Comment by Veyfeyken** [comment]
This is fixed for most but unfortunately not for all icons yet. The accessibility icon itself for example:

```html
<span class="fa fa-universal-access u-text-primary">
::before
<span class="u-screen-reader-only">accessibility note</span>
</span>
```
